subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -32,7 +32,7 @@ LOG_MODULE_REGISTER(net_wifi_shell, LOG_LEVEL_INF);
 				NET_EVENT_WIFI_DISCONNECT_RESULT)
 
 static struct {
-	const struct shell *shell;
```

Comment:
Why do we change that?

Reply:
See commit log https://github.com/nrfconnect/sdk-zephyr/pull/876/commits/a953138b89cdea772060a38ab87890e2b0bd202e